### DISCUSSION TITLE
What is the unit of `stats. storage_size` in `/api/index` response

### DISCUSSION THREAD

**Discussion by deepjyoti30** [zinclabs/zinc#602] [Q&A] [answered]
As the question suggests, the `/api/index` returns the list of indices. Inside each list object there is a `stats` object inside where there is a field `storage_size`. There is no mention of what the unit of this value is.

Just want to know if this value is in `bytes` or `bits`?

**Comment by hengfeiyang** [comment, marked as answer]
bytes
